### ISSUE TITLE
`make build` does not make sense

### ISSUE THREAD

**Issue by maskshell** [openresty/openresty-packaging#71]
`README.md` says, "... to build everything from scratch, just run `make build`".

However, the behavior of `make build` does not make sense.

The main content of `make build`:

```
.PHONY: build
build: zlib-build pcre-build openssl-build \
        openssl-debug-build openresty-build openresty-debug-build \
        openresty-valgrind-build \
        lemplate-build test-nginx-build
```

However, in almost every sub-build, there is `rm -f *.deb`, which makes `make build` end up with only the deb package of the last build as far as outputting deb packages.

**Comment by xiaocang** [comment]
Yep, you're right, but you can put an shell script named `upload` locally, which will be called after the packaging is finished
